### DISCUSSION TITLE
How to use a literal pipe character in statfmt?

### DISCUSSION THREAD

**Discussion by mkacct** [gokcehan/lf#1812] [Q&A] [answered]
Reading the docs on [`statfmt`](https://github.com/gokcehan/lf/blob/master/doc.md#statfmt-string-default-03336mp0330m-c-u-g-s-t---l) (wherein the pipe character separates sections) and experimenting a little, I can't seem to determine how to escape a `|` character. How can this be done?

**Comment by joelim-work** [comment]
Literal `|` is not supported currently because it didn't appear to be useful. You can have a look at the discussion in #1337 for more details.

What is your use case, and is it not possible to use some other Unicode character that looks similar?

**Reply by mkacct** [reply, marked as answer]
Good to know, I'll just use `│` instead
